### PR TITLE
Fix dev

### PR DESCRIPTION
**Pull request by zb32**
## PR checklist

- [ X ] This comment contains a description of changes (with reason).
   Update module directory structure to match nf-core V2.6 and update documentation. 
- [ X ] Make sure your code lints (`nf-core lint`).
- [X ] Ensure the test suite passes (`nextflow run . -profile test,docker --outdir <OUTDIR>`).
- [X ] Usage Documentation in `docs/usage.md` is updated.
- [ X] `README.md` is updated (including new tool citations and authors/contributors).

### CODE REVIEW

**Review by priyanka-surana — CHANGES_REQUESTED** [review]
Include everything till the BUSCO subworkflow (#36).
1. README.md - Address all TODOs: https://github.com/sanger-tol/readmapping/blob/main/README.md
2. Add docs/parameters.md using `nf-core schema docs --help`: https://github.com/sanger-tol/readmapping/blob/main/docs/parameters.md
3. Add docs/output.md: https://github.com/sanger-tol/insdcdownload/blob/dev/docs/output.md

File: README.md [hunk 1]
```diff
@@ -17,22 +17,35 @@
...
 
 <!-- TODO nf-core: Add full-sized test dataset and amend the paragraph below if applicable -->
 
-On release, automated continuous integration tests run the pipeline on a full-sized dataset on the AWS cloud infrastructure. This ensures that the pipeline runs on AWS, has sensible resource allocation defaults set to run on real-world datasets, and permits the persistent storage of results to benchmark between pipeline releases and other analysis sources. The results obtained from the full-sized test can be viewed on the [nf-core website](https://nf-co.re/blobtoolkit/results).
+On release, tests run the pipeline on a dataset on the Wellcome Sanger Institute HPC farm with Sigularity containers. This ensures that the pipeline runs, has sensible resource allocation defaults set to run on real-world datasets, and permits the persistent storage of results to benchmark between pipeline releases and other analysis sources. The results obtained from the test can be viewed on the [sanger-tol website](https://github.com/sanger-tol/blobtoolkit).
```
Comment (priyanka-surana) [inline README.md:26]: We do not have a place where the test results can be viewed yet. I think it is best if we delete the last sentence 
> The results obtained from ...

File: README.md [hunk 2]
```diff
@@ -63,7 +76,7 @@ On release, automated continuous integration tests run the pipeline on a full-si
 
 ## Documentation
 
-The nf-core/blobtoolkit pipeline comes with documentation about the pipeline [usage](https://nf-co.re/blobtoolkit/usage), [parameters](https://nf-co.re/blobtoolkit/parameters) and [output](https://nf-co.re/blobtoolkit/output).
+The nf-core/blobtoolkit pipeline comes with documentation about the pipeline [usage](https://github.com/sanger-tol/blobtoolkit/blob/main/docs/usage.md), [parameters](https://github.com/sanger-tol/blobtoolkit/blob/main/docs/parameters.md) and [output](https://github.com/sanger-tol/blobtoolkit/blob/main/docs/output.md).
 
 ## Credits
```
Comment (priyanka-surana) [inline README.md:81]: Need to update credits.
1. Change from `nf-core/blobtoolkit` to `sanger-tol/blobtoolkit`
2. Add your name to the list as well.
3. Give credit to Richard Challis for design/original pipeline and Sujai Kumar for maintaining + Link to the original blobtoolkit
4. Add Matthieu and me
5. Also add a sentence on how Alexander was supported by GSoC
6. Remove TODO

**Review by priyanka-surana — CHANGES_REQUESTED** [review]
1. The usage docs refers to `nf-core/blobtoolkit` and that link, please update
2. In general do a search for similar terms/links throughout the pipeline and update.

File: .github/workflows/ci.yml [hunk 3]
```diff
@@ -20,14 +20,9 @@ jobs:
...
-        include:
-          # Test pipeline minimum Nextflow version
-          - NXF_VER: "21.10.3"
-            NXF_EDGE: ""
-          # Test latest edge release of Nextflow
-          - NXF_VER: ""
-            NXF_EDGE: "1"
+        NXF_VER:
+          - "21.10.3"
+          - "latest-everything"
```
Comment (gq1) [inline .github/workflows/ci.yml:25]:
```suggestion
          - ""
```

File: .github/workflows/ci.yml [hunk 4]
```diff
@@ -20,14 +20,9 @@ jobs:
...
-        # Nextflow versions
-        include:
-          # Test pipeline minimum Nextflow version
-          - NXF_VER: "21.10.3"
-            NXF_EDGE: ""
-          # Test latest edge release of Nextflow
-          - NXF_VER: ""
-            NXF_EDGE: "1"
+        NXF_VER:
+          - "21.10.3"
```
Comment (priyanka-surana) [inline .github/workflows/ci.yml:24]: I would update this to `22.10.1` at least, but stick to a stable version.
https://github.com/nextflow-io/nextflow/releases

File: README.md [hunk 5]
```diff
@@ -1,38 +1,51 @@
...
 [![run with conda](http://img.shields.io/badge/run%20with-conda-3EB049?logo=anaconda)](https://docs.conda.io/en/latest/)
 [![run with docker](https://img.shields.io/badge/run%20with-docker-0db7ed?logo=docker)](https://www.docker.com/)
 [![run with singularity](https://img.shields.io/badge/run%20with-singularity-1d355c.svg)](https://sylabs.io/docs/)
-[![Launch on Nextflow Tower](https://img.shields.io/badge/Launch%20%F0%9F%9A%80-Nextflow%20Tower-%234256e7)](https://tower.nf/launch?pipeline=https://github.com/nf-core/blobtoolkit)
+[![Launch on Nextflow Tower](https://img.shields.io/badge/Launch%20%F0%9F%9A%80-Nextflow%20Tower-%234256e7)](https://tower.nf/launch?pipeline=https://github.com/sanger-tol/blobtoolkit)
 
 [![Get help on Slack](http://img.shields.io/badge/slack-nf--core%20%23blobtoolkit-4A154B?logo=slack)](https://nfcore.slack.com/channels/blobtoolkit)
 [![Follow on Twitter](http://img.shields.io/badge/twitter-%40nf__core-1DA1F2?logo=twitter)](https://twitter.com/nf_core)
 [![Watch on YouTube](http://img.shields.io/badge/youtube-nf--core-FF0000?logo=youtube)](https://www.youtube.com/c/nf-core)
```
Comment (priyanka-surana) [inline README.md:17]: Either remove Slack, Twitter and YouTube or update it to the correct links. It is all pointing to `nf-core` at this point.

File: assets/email_template.txt [hunk 6]
```diff
@@ -6,7 +6,6 @@
                                         `._,._,'
   nf-core/blobtoolkit v${version}
```
Comment (priyanka-surana) [inline assets/email_template.txt:7]: This should be `sanger-tol`

File: conf/modules.config [hunk 7]
```diff
@@ -12,11 +12,13 @@
...
-        mode: params.publish_dir_mode,
-        saveAs: { filename -> filename.equals('versions.yml') ? null : filename }
-    ]
+    withName: '.*:.*:COVERAGE_STATS:MOSDEPTH' {
+        publishDir = [
+            path: { "${params.outdir}/blobtoolkit" },
+            mode: params.publish_dir_mode,
+            saveAs: { filename -> filename.equals('versions.yml') ? null : filename }
+        ]
+    }
```
Comment (priyanka-surana) [inline conf/modules.config:21]: Was this not in your original `mosdepth` PR?

File: docs/usage.md [hunk 8]
```diff
@@ -21,34 +13,30 @@ You will need to create a samplesheet with information about the samples you wou
...
 ```console
-sample,fastq_1,fastq_2
-CONTROL_REP1,AEG588A1_S1_L002_R1_001.fastq.gz,AEG588A1_S1_L002_R2_001.fastq.gz
-CONTROL_REP1,AEG588A1_S1_L003_R1_001.fastq.gz,AEG588A1_S1_L003_R2_001.fastq.gz
-CONTROL_REP1,AEG588A1_S1_L004_R1_001.fastq.gz,AEG588A1_S1_L004_R2_001.fastq.gz
+sample,datatype,datafile
+sample1, hic, hic.cram
+sample2, illumina, illumina.cram
+sample2, illumina, illumina.cram
 ```
```
Comment (priyanka-surana) [inline docs/usage.md:20]: No spaces in comma separated files. This will cause errors in processing.

File: docs/usage.md [hunk 9]
```diff
@@ -21,34 +13,30 @@ You will need to create a samplesheet with information about the samples you wou
...
-CONTROL_REP3,AEG588A3_S3_L002_R1_001.fastq.gz,AEG588A3_S3_L002_R2_001.fastq.gz
-TREATMENT_REP1,AEG588A4_S4_L003_R1_001.fastq.gz,
-TREATMENT_REP2,AEG588A5_S5_L003_R1_001.fastq.gz,
-TREATMENT_REP3,AEG588A6_S6_L003_R1_001.fastq.gz,
-TREATMENT_REP3,AEG588A6_S6_L004_R1_001.fastq.gz,
+sample,datatype,datafile
+sample1, hic, hic.cram
+sample2, illumina, illumina.cram
+sample3, ont, ont.cram
 ```
```
Comment (priyanka-surana) [inline docs/usage.md:33]: Same here. No spaces please. During processing, spaces will get read as part of the file name or data type and cause errors. Does the test samplesheet have spaces?

**Review by priyanka-surana — CHANGES_REQUESTED** [review]
Small changes. Mostly looks good 👍

**Review by priyanka-surana — APPROVED** [review]
I think this is all good. I would recommend adding your name as the author as well in `nextflow.config`. Let's get one more approval before we merge.

File: nextflow.config [hunk 10]
```diff
@@ -163,7 +163,7 @@ dag {
 
 manifest {
     name            = 'sanger-tol/blobtoolkit'
-    author          = 'Alexander Ramos'
+    author          = 'Alexander Ramos and Zaynab Butt'
```
Comment (priyanka-surana) [inline nextflow.config:166]:
```suggestion
    author          = 'Alexander Ramos, Zaynab Butt'
```

**Pull request merged**